test/empty-set.js

```diff
@@ -7,7 +7,7 @@ var patterns = [
   '# comment',
   ' ',
   '\n',
-  'just doesnt happen to match anything so this is a control'
+  'just doesn't happen to match anything so this is a control'
```

Comment:
Looks like the single quote here needs to be escaped, causing a syntax error otherwise.

Reply:
Sorry about that. I will close this request.